src/content/protocols/stakewise-v3/ethereum.md

```diff
@@ -0,0 +1,250 @@
+---
+chain: "ethereum"
+stage: 0
+reasons: []
+risks: ["L", "H", "H", "H", "M"]
+author: ["eliasbourgon"]
+submission_date: "1970-01-01"
+publish_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+StakeWise v3 is a liquid-staking protocol that lets users stake ETH and earn rewards while keeping their assets liquid. It tokenizes staking positions into transferable ERC-20–compatible tokens and introduces a modular “vault” architecture that allows anyone , from individuals to DAOs, to create custom staking strategies under unified security and accounting rules.
+
+# Ratings
+
+## Chain
+
+Stakewise V3 is deployed on Ethereum and Gnosis. This review is based on the Ethereum mainnet deployment.
+
+> Chain score: Low
+
+## Upgradeability
+
+StakeWise v3 relies on a range of upgradeable components, both at the bytecode level and through owner-controlled configuration parameters. Several contracts, such as the vault implementations, the OsToken suite, the Keeper, and the various factories, are deployed behind upgradeable proxy patterns or expose administrative entry points allowing their logic to be replaced. The entities with upgrade permissions (primarily the StakeWise DAO multisig, the Stakewise Multisig #1, and #2) therefore retain the ability to modify how core mechanisms behave, including the handling of deposits, redemptions, reward accounting, validator assignments and the overall minting and burning logic for OsToken.
+
+In addition to full logic upgrades, a large number of parameter changes can be performed without modifying bytecode. These parameters include fee destinations, blocklist administration, validator roots, keeper thresholds, reward delays, liquidation settings, vault capacities, the controller of OsToken, and the set of redeemable positions in the redeemer. Because these parameters influence how assets flow through the system, how user positions evolve over time and how rewards are computed or distributed, changes made through these permissions can materially alter the behaviour users experience, even if the underlying logic contracts remain the same.
```

Comment:
Good paragraph, can you specify exactly whether those may lead to loss of user funds or loss of unclaimed yield and why? Who owns the permissions?